### ISSUE TITLE
Crashing Due to Too Many CPU Cores.

### ISSUE THREAD

**Issue by PixelCrusader** [emoose/OutRun2006Tweaks#32]
I'm not an expert, so I apologize if my descriptions are insufficient. That said, I've noticed that OutRun 2006 doesn't play nice with CPUs that are too modern, likely due to the game not being designed to handle so many processor cores at once. What will usually happen is that the game will launch and start a race without issues, but when the race is over and the player tries to load back into the menus, the game will crash during the loading screen.

In the past, I was able to get around this problem by using the Microsoft Application Compatibility Toolkit and applying the SingleProcAffinity fix to fool the game into thinking my CPU (Intel Core i9 10900K) had less cores than it really did. This worked fine on the vanilla game or FXT, but when I apply the OutRun2006Tweaks mod to the game, it starts to stutter and drop frames. It looks like only one of my cores is insufficient to run the game and the mod at full speed.

I have to disable the SingleProcAffinity fix to get the game to run without dropping frames, but doing that means that the game goes back to crashing at the end of races. I can work around this by running Steam with the processor affinity set to only use CPUs 0-7, but this is an inelegant solution as I have to do this only for OutRun and undo it when I want to play anything else.

So I guess what I'm trying to say is that it would be really nice if you would be able to fix the game so that it wouldn't be affected by this problem anymore. Thanks for the work so far. Having full rumble support and the pegasus sound problem fixed are really great to enjoy.

**Comment by scheeseman486** [comment]
I have a i5 14600K and am also running into random crashes, either during the opening splashes or when coming back from a race.

**Comment by PixelCrusader** [comment]
I've downloaded and tried out the v0.6 patch today, and it looks like the problem I brought up has been addressed with v0.5.1. At least on my end, the game is now running at full speed, without crashing after the race. I no longer have to adjust the processor affinity settings. Thanks much for solving it. Also thanks for fixing the shadows on the cars. The 360 Spider is my favorite and it's great to no longer have to see the shadows break.
